uiowa/uiowa
# Remove unused drush config and uninstall automated_cron.

Resolves #4839 
Resolves #4841 

Drush no longer caches anything and doesn't use the drush-paths-cache-directory config and/or environment variables. This PR removes that unused code. It also removes some Drush config that we weren't using. 

It also uninstalls automated cron.

Cron jobs have been set up on all application dev/test environments. Prod can be configured once this is deployed.

# How to test

- Review docs at bottom on https://wiki.uiowa.edu/display/drupaldev/Acquia+Applications re: setting up new applications.
- Log in to Acquia Cloud.
- Go to each D9 application dev/test env and inspect the scheduled tasks.
- Go to uiowa in AC interface. SSH in to uiowa dev in two separate terminals. Change directory up one level, i.e.`cd ..`
- Start a code checkout on uiowa dev for the develop-build branch.
- Simultaneously run `./vendor/bin/blt ume cr` and `/.vendor/bin/blt ume cron` in the terminal sessions.
- There should be no errors related to Drush cache in the AC task output or the terminals. The AC task is currently failing for config reasons and the terminal running `cron` will show errors for empty databases like this:
```
Command cron was not found. Drush was unable to query the database. As a result, many commands are unavailable. Re-run your command with --debug to see relevant log messages.
```

Those are expected.

- Sync a site and see that automated_cron is uninstalled and the config deleted.

*This is not exactly the same thing as prod and ultimately that is where this needs to be tested.
